src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "CloudWatch fallback"
+metaDescription: This doc guides you through shipping your telemetry data with Cloudwatch as a fail-safe option.
+freshnessValidatedDate: never
+---
+
+You can ship New Relic Lambda monitoring data in a few different ways depending on your needs. The option that provides the strongest performance of your function and the strongest fail-safe is the CloudWatch fallback option. This means that you will ship telemetry to New Relic through the agent and use the extension to enhance the telemetry collection but, if there's a problem with the extension, you will still receive logs and payloads in CloudWatch.  
+
+While this offers the strongest fail-safe, it's important to note that the AWS CloudWatch service can generate a lot of data. Keep data ingest in mind when thinking about costs and which New Relic pricing plan you choose. 
+
+Benefits of the CloudWatch fallback:
+
+* **Independent execution**: Captures all logs from CloudWatch independently of your function's execution. There's less chance it will interfere with the normal operation, or impact the invocation duration of your function.
+* **Works with secured networks**: Useful for functions in a VPC that can't have outbound traffic to New Relic.
+* **Enables enhanced monitoring:  Needed for other forms of infra and database telemetry. To learn more, see our [Amazon RDS Enhanced Monitoring integration docmentation](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rds-enhanced-monitoring-integration/).
+
+## CloudWatch fallback [#fallback]
+
+Even with those best practices in place, having a fallback can always be a good plan of action. There are three ways you can implement a CloudWatch fallback:
+
+    1. **Enable the extension enabled for payloads and disable the extension for logs**: In this scenario, if the extension fails, payloads will be sent through CloudWatch but CloudWatch will always send logs. 
+    2. **Extension enabled for payloads and logs**: In this scenario, if the extension fails, CloudWatch will send payloads. 
```

Comment:
> Payloads and logs via our extension with payloads (fallback) via CloudWatch

Reply:
@keegoid-nr should it be:

Should it be:

"In this scenario, payloads and logs are shipped through the extension and payloads through CloudWatch in the event the extension fails."